programs/svm-spoke/src/instructions/deposit.rs

```diff
@@ -21,15 +21,7 @@ use crate::{
 
 #[event_cpi]
 #[derive(Accounts)]
-#[instruction(
-    depositor: Pubkey,
-    recipient: Pubkey,
-    input_token: Pubkey,
-    output_token: Pubkey,
-    input_amount: u64,
-    output_amount: u64,
```

Comment:
I was going to change `output_amount` type, but realized we only need args up to `input_token` here